db/pre-re/item_db.conf

```diff
@@ -66942,7 +66952,7 @@ item_db: (
 	Job: 0x00000400
 	Upper: 8
 	BuyingStore: true
-	Script: <" //madoheal rand(300,400),0; ">
+	Script: <" if(checkmount() == MOUNT_MADO) itemheal rand(300,400), 0; ">
```

Comment:
I remember I read a blog in rathena saying ... 3rd job never exist in pre-renewal
https://rathena.org/board/blog/23/entry-87-pre-renewal-rathena/
if we want to follow it officially, yeah pre-renewal shouldn't even have any 3rd job stuff
I don't play official server anymore so don't ask  me about official things